boards/nxp/frdm_rw612/doc/index.rst

```diff
@@ -57,7 +57,7 @@ Supported Features
 | OS_TIMER  | on-chip    | os timer                          |
 +-----------+------------+-----------------------------------+
 | PM        | on-chip    | power management; uses SoC Power  |
-|           |            | Modes 1 and 2                     |
+|           |            | Modes 1, 2 and 3                  |
```

Comment:
Be aware, this table is removed in https://github.com/zephyrproject-rtos/zephyr/pull/87797, and will be auto-generated.  You could add a new section in this doc detailing the power modes supported.